Test/Merge-Migration.Tests.ps1

```diff
@@ -745,4 +745,33 @@ Describe 'Merge-Migration when removing and re-adding the same table across migr
         $ops[1].ColumnName.Count | Should Be 1
         $ops[1].ColumnName[0] | Should Be 'TestUserID2'
     }
-}
\ No newline at end of file
+}
+
+Describe 'Merge-Migration.when table and column have same name and the table is renamed' {
+    $result = Invoke-MergeMigration {
+        New-MigrationObject 'one' {
+            Add-Table -Name 'OldName' {
+                int 'id'
+            }
+        }
+        New-MigrationObject 'two' {
+            Update-Table 'OldName' -AddColumn {
+                tinyint 'OldColumn' -NotNull
+            }
+        }
+        New-MigrationObject 'three' {
+            Rename-Object -Name 'OldName' -NewName 'NewName'
+        }
+        New-MigrationObject 'four' {
+            Add-Table -Name 'OldColumn' {

```

Comment:
Might be helpful to add comment here to indicate table added with the duplicate name.